src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -337,12 +337,13 @@ public static function getService(): array
 	/**
 	 * Return the ActivityPub profile of the given user
 	 *
-	 * @param int $uid User ID
+	 * @param int  $uid     User ID
+	 * @param bool $limited If limited, only the basic information is returned
 	 * @return array with profile data
 	 * @throws HTTPException\NotFoundException
 	 * @throws HTTPException\InternalServerErrorException
 	 */
-	public static function getProfile(int $uid): array
+	public static function getProfile(int $uid, bool $limited = false): array
```

Comment:
Please use a positive argument name, it will save us many `!` operators in this method and in all the calls.
```suggestion
	public static function getProfile(int $uid, bool $full = true): array
```